### ISSUE TITLE
Should we add size check to tests

### ISSUE THREAD

**Issue by balloob** [reactstrap/reactstrap#475]
4.6.0 introduced an issue with the bundle size that went unnoticed. As a comment on PR #474 which fixes the size issue, @nathancahill suggested we make bundle size part of the build process.

Instead of the build process, I think we should add it as a test. That way when a PR breaks the build, CI will fail. If it's part of the build process, we will only notice something is broken when we try to do a release and have to hunt down which PR broke it.

It would require us to do a prod build before running the tests. This currently takes ~13s on my machine.

Initially I would suggest we set the limit to 150kB. That's ~50% above the current size. This should catch increases introduced by both #399 and #433.

This wouldn't be a hard limit either. We just need some number to catch big increases.

**Comment by joseluisq** [comment]
:+1:

**Comment by TheSharpieOne** [comment]
This is incorrect, the issue was not caused by #433. Reverting that change will break reactstrap when be using without treeshaking, which is a major use case at this point. 
Per best practices, NEVER point to a precompiled version. This is that yellow warning which you also break things which are not webpack.

**Comment by balloob** [comment]
I see your react@15.6.1 and I raise you https://unpkg.com/react@16.0.0-alpha.13/ 😉

**Comment by TheSharpieOne** [comment]
I raise you cjs vs umd in react 16.
